src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -33,25 +40,115 @@ exports.handler = async function (event, context, callback) {
         region,
         secretId,
       });
+      let recipientType;
+      let recipients;
+      // if it greater then 2 days but less then 4 days
+      if (
+        data.secSinceMmdlSigned > 48 * 3600 &&
+        data.secSinceMmdlSigned < 48 * 2 * 3600
+      ) {
+        recipientType = "emailRecipientsA";
+        recipients = emailRecipientsA;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned > (48 * 3600)) && (data.secSinceMmdlSigned < ((48 * 2) * 3600))":
+            data.secSinceMmdlSigned > 48 * 3600 &&
+            data.secSinceMmdlSigned < 48 * 2 * 3600,
+        });
+      }
+      // if it is greater then 4 days
+      else if (data.secSinceMmdlSigned > 48 * 2 * 3600) {
+        recipientType = "emailRecipientsB";
+        recipients = emailRecipientsB;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned > (48 * 3600))":
+            data.secSinceMmdlSigned > 48 * 3600,
+        });
+      }
+      // if it is less then 2 days
+      else {
+        recipientType = "emailRecipients";
+        recipients = emailRecipients;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned < (48 * 3600))":
+            data.secSinceMmdlSigned < 48 * 3600,
+        });
+      }
 
       // you can also use the data.programType value here if needed "MAC" | "HHS" | "CHP"
-      const params = getRecordDoesNotExistParams({
-        emailRecipients,
-        sourceEmail,
-        id: data.id,
-      });
+      let params;
+      if (!isProgramTypeChp) {
+        //for non chip
+        if (recipientType == "emailRecipients") {
+          params = getEmailParams({
+            emailRecipients: recipients,
+            sourceEmail: sourceEmail,
+            id: data.id,
+            Template: Templates.SendNoMatchTemplate
+          });
+        }else{
+          params = getEmailParams({
+            emailRecipients: recipients,
+            sourceEmail: sourceEmail,
+            id: data.id,
+            Template: Templates.SendNoMatchTemplateAB
+          });
+        }
+      }else{
+        // for chip
+        if (recipientType == "emailRecipients") {
+          params = getEmailParams({
+            emailRecipients: recipients,
+            sourceEmail: sourceEmail,
+            id: data.id,
+            Template: Templates.SendNoMatchTemplateChp
+          });
+        }else{
+          params = getEmailParams({
+            emailRecipients: recipients,
+            sourceEmail: sourceEmail,
+            id: data.id,
+            Template: Templates.SendNoMatchTemplateChpAB
+          });
```

Comment:
the only thing actually changing here is the template. we can prob use the logic from the above comment to get the recipients and then just call this function once.